src/main/java/edu/cornell/kfs/pdp/service/impl/CuPendingTransactionServiceImpl.java

```diff
@@ -962,7 +957,14 @@ protected boolean generateEntriesPaymentRequest(PaymentRequestDocument preq, Lis
 
           
         }
-
+        
+        if (CollectionUtils.isNotEmpty(preq.getItems())) {
```

Comment:
Based on the description for this user story, I think the cancel-revisions logic should be executed when "summaryAccounts" is non-empty. Can this "if" condition be updated accordingly, or be removed and have its internal code moved into the block above that already tests the "summaryAccounts" list?

Reply:
I will move this code further up so it is in if statement that checks if summaryAccounts is populated.  I do think the if statement checking preq.getItems() is needed because when looking at PurapAccountRevisionService.cancelPaymentRequestAccountRevisions, it will error of getItems is null, and won't do anything if it is empty